docs/proposals/pepxxx_wheel_variant_support.md

```diff
@@ -166,6 +746,8 @@ A prototype implementation has been developed, demonstrating:
 
 ## Rejected Ideas
 
+### Alternative approaches
```

Comment:
The most important alternative is to make platform tags GPU aware. This was rejected due to the combinatorial explosion, the platform tags already being complex and the GPUs changing relative quickly (unlike OS, CPU architecture and libc kinds, which barely changed in the last decade)